### PR TITLE
update mesh conversion: extract cells by material id

### PR DESCRIPTION
**Pull request by vlukes**

### CODE REVIEW

**Comment by rc** [comment]
Thanks, useful addition!

Some remarks:
- Usually, the short options have a single letter - I suggest removing ``-pm``, this is not a very common operation.
- Use, please, ``--save-per-mat`` instead of ``--save_per_mat``
- Most os.path imports in sfepy are ``import os.path as op``
- The help message could be more verbose, something like:

    extract cells by material id and save them into a separate mesh file with a name based on `filename_out` and the id; preserves original mesh vertices

File: doc/preprocessing.rst [hunk 1]
```diff
@@ -370,6 +370,15 @@ argument to `convert_mesh.py` script::
...
 
+In order to extract the cells by the physical groups use the conversion script
+with ``--save_per_mat`` (``-pm``) argument::
+
+  script/convert_mesh.py -pm screwdriver_full.vtk screwdriver.vtk
+
+It produces `screwdriver_matid_1.vtk` and `screwdriver_matid_2.vtk` files
+containing only the cells of a given physical group and all vertices of the
+original mesh.
+
```
Comment (rc) [inline doc/preprocessing.rst]: It saves those meshes in addition to the output mesh - maybe mention that?

Comment (vlukes) [reply doc/preprocessing.rst]: Yes, I added the comment to this.

**Comment by vlukes** [comment]
The remarks are taken into account in the updated commit.

**Pull request merged**